core-web/libs/dotcms-scss/shared/_common.scss

```diff
@@ -38,3 +38,6 @@ $notification-dot-badge-size: 0.625rem;
 // Content drive animations
 $content-drive-animation-speed: 0.3s;
 $content-drive-animation-ease: ease-in-out;
+
+// Content drive dropzone
+$content-drive-dropzone-background-color: rgba(246, 249, 252, 0.8);
```

Comment:
All our hovers or "active" elements should be the same, we should not be creating vars for each.

Reply:
Im waiting for a response from @melissarojas-dotcms of what are we going to do with that color

I did this because I was reusing it, but is not the case, I will remove it